### PR TITLE
Add Tim Downey as a CLI reviewer

### PR DESCRIPTION
**Pull request by tcdowney**
Hello, I would like to formally be a Reviewer for the CF CLI. Here are some of my past contributions:

* Commits: https://github.com/cloudfoundry/cli/commits?author=tcdowney
* PRs: https://github.com/cloudfoundry/cli/pulls?q=is%3Apr+author%3Atcdowney+is%3Aclosed

### CODE REVIEW

**Comment by beyhan** [comment]
We need two thumps up or approvals from the CLI approvers for this pr before it can be merged because the CLI are has more than four approvers.

**Comment by Gerg** [comment]
@cloudfoundry/wg-app-runtime-interfaces-cli-approvers

**Review by moleske — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Comment by stephanme** [comment]
The Reviewer requirements are met. Welcome @tcdowney in CLI area of ARI WG.

**Pull request merged**
